From e2dc0c834fa17f462e0e0d80b2be919c4e788336 Mon Sep 17 00:00:00 2001
Message-ID: <e2dc0c834fa17f462e0e0d80b2be919c4e788336.1732384228.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1732384228.git.sharlatanus@gmail.com>
References: <cover.1732384228.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 21 Nov 2024 17:23:34 +0000
Subject: [PATCH 39/46] gnu: python-statmorph: Update to 0.6.0.

* gnu/packages/astronomy.scm (python-statmorph): Update to 0.6.0.
[arguments]<test-flags>: Enable all tests as issue is resolved upstream.
[propagated-inputs]: Remove python-matplotlib.

Change-Id: I48b71e590b9c4da4f09ab578767c05890110e8d4
---
 gnu/packages/astronomy.scm | 28 ++--------------------------
 1 file changed, 2 insertions(+), 26 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 76910307c5..a83cee194b 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -4976,43 +4976,19 @@ (define-public python-spisea
 (define-public python-statmorph
   (package
     (name "python-statmorph")
-    (version "0.5.7")
+    (version "0.6.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "statmorph" version))
        (sha256
-        (base32 "06rbwrqlvdzn9innx05kjjmm4mmacd8kwwsfdf5idbwzby5ny5lw"))))
+        (base32 "1575vmk4kl06yxyid7mb1r7llbrmraagvx2b0h5x335ha7irnkjk"))))
     (build-system pyproject-build-system)
-    (arguments
-     (list
-      #:test-flags
-      ;; XXX: Skip most of the tests for now, see
-      ;; <https://github.com/vrodgom/statmorph/issues/11>.
-      #~(list "-k" (string-append "not test_asymmetric"
-                                  " and not test_bright_pixel"
-                                  " and not test_catastrophic"
-                                  " and not test_empty_gini_segmap"
-                                  " and not test_full_gini_segmap"
-                                  " and not test_full_segmap"
-                                  " and not test_insufficient_data"
-                                  " and not test_make_figure"
-                                  " and not test_masked_centroid"
-                                  " and not test_merger"
-                                  " and not test_missing_arguments"
-                                  " and not test_negative_source"
-                                  " and not test_no_psf"
-                                  " and not test_psf"
-                                  " and not test_random_noise"
-                                  " and not test_small_source"
-                                  " and not test_tiny_source"
-                                  " and not test_weightmap"))))
     (native-inputs
      (list python-pytest
            python-setuptools))
     (propagated-inputs
      (list python-astropy
-           python-matplotlib
            python-numpy
            python-photutils
            python-scikit-image
-- 
2.46.0

